src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -36,6 +36,7 @@ const BottomBox = ({
   children: React.ReactNode;
 }) => (
   <div
+    role="note"
```

Comment:
This was made, so that screenreader should not "join" different texts together when reading randomly.